test/src/tests/generated/additional-properties/pets.go

```diff
@@ -31,6 +33,18 @@ func NewPetsClientWithBaseURI(baseURI string) PetsClient {
 
 // CreateAPInProperties create a Pet which contains more properties than what is defined.
 func (client PetsClient) CreateAPInProperties(ctx context.Context, createParameters PetAPInProperties) (result PetAPInProperties, err error) {
+	// Not necessary to perform this check as nothing will be instrumented if it is false, but
```

Comment:
Hmm I'm wondering if this comment should be removed.  It adds two lines per API making the files longer and it's likely nobody will ever read it.

Reply:
well my reason was just not to make people think they need to do  an `isEnabled()` check. If you think is overkill I can remove it

Reply:
and let me know if there is anything else

Reply:
thought about it a little, and yea seems a little overkill, makes more sense maybe to add this note in the go-autorest functions directly, so removing it